src/CMakeLists.txt

```diff
@@ -52,6 +52,12 @@ add_library(migraphx
     normalize_attributes.cpp
     normalize_ops.cpp
 )
+file(WRITE include/migraphx/version.h 
```

Comment:
This shouldn't write to the source tree. Instead, there should be a version.h.in and this file should be configured by cmake.

Reply:
Fixed